### PR TITLE
doc(.github): add pull request and issue templates

### PR DESCRIPTION
**Pull request by arschles**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+Please provide a summary of this issue.
+
+## If You Are Reporting a Bug
+
+If you're reporting a bug, please provide a detailed description of the problem.
+
+## If You Are Submitting a Feature Request
+
+If you're submitting a feature request, please provide as much details on what you'd like to see and 1 or more use cases. The more detail we have, the more quickly and efficiently we can discuss, decide and implement the feature.
+If this is a bug: provide a description of the problem
```
Comment (bacongobbler) [inline .github/ISSUE_TEMPLATE]: looks like this is repeated above on line 7

File: .github/ISSUE_TEMPLATE [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## Everything Else
+
+If you're not submitting a bug or feature request, we still want to hear from you! Please feel free to write us and we'd love to have a conversation. As always, constructive conversation and details, where appropriate, are appreciated.
+
+# For Bugs: Replication Steps
```
Comment (bacongobbler) [inline .github/ISSUE_TEMPLATE]: this might be better to place under the first header, "If you are reporting a bug"

File: .github/ISSUE_TEMPLATE [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+Also, please provide a description of the desired result after the tester completes the above steps.
+
+1. The app called "abcd" should be deployed
+2. `kubectl get pod --namespace=abcd` should show 2 pods running
+3. etc...
+
+# Related Issues
+
+Do any issues in this repository relate to this one? If so, refer to them here and explain why they relate
```
Comment (bacongobbler) [inline .github/ISSUE_TEMPLATE]: Do any issues or pull requests*

Sometimes a PR can reference another PR (i.e. chart changes)

File: .github/ISSUE_TEMPLATE (hunk 1 above)
Comment (arschles) [inline .github/ISSUE_TEMPLATE]: Thanks, fixed in https://github.com/arschles/deis-builder/commit/dc1b00d7ac2ff7ac35450fdf367375384a90c3ad

File: .github/ISSUE_TEMPLATE (hunk 2 above)
Comment (arschles) [inline .github/ISSUE_TEMPLATE]: Good idea. I moved it in Thanks, fixed in https://github.com/arschles/deis-builder/commit/dc1b00d7ac2ff7ac35450fdf367375384a90c3ad

File: .github/ISSUE_TEMPLATE [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# Summary
+
+Please provide a summary of this issue.
+
+## If You Are Reporting a Bug
+
+If you're reporting a bug, please provide a detailed description of the problem.
+
+### Replication Stps
```
Comment (kmala) [inline .github/ISSUE_TEMPLATE]: It should be Steps

Comment (arschles) [inline .github/ISSUE_TEMPLATE]: Fixed in https://github.com/deis/builder/pull/233/commits/c9e4cdef8eeabe2390d089c8cdbed4c53dfc0024

**Pull request merged**